tools/sup-network/Makefile

```diff
@@ -0,0 +1,50 @@
+ifneq (${HAB_SUP_TEST_NETWORK_LOG_SERVICE},)
+	log_service := ${HAB_SUP_TEST_NETWORK_LOG_SERVICE}
+else
+	log_service := rando
+endif
+
+ifneq (${HAB_SUP_TEST_NETWORK_SIZE},)
+	peers := ${HAB_SUP_TEST_NETWORK_SIZE}
+else
+	peers := 3
+endif
+
+HAB_SUP_TEST_NETWORK_PEER_NAME ?= rando_1
+
+up: build CTL_SECRET
+	@docker build -t hab-sup-network .
```

Comment:
In general, I prefer to only apply the `@` in front of commands that are for output themselves like `echo`, because it's redundant. For these kinds of commands, I find it instructive to have them printed to better know what's going on.

Reply:
Sure, I can update this.